subsys/bluetooth/shell/bt.c

```diff
@@ -3739,6 +3739,41 @@ static int cmd_bondable(const struct shell *sh, size_t argc, char *argv[])
 	return 0;
 }
 
+#if defined(CONFIG_BT_BONDABLE_PER_CONNECTION)
+static int cmd_conn_bondable(const struct shell *sh, size_t argc, char *argv[])
+{
+	int res;
```

Comment:
```suggestion
	int err;
```

It's more common to use `err` rather than `res` in this file

Reply:
Updated. However, it seems that the previous code did not follow this rule.

Reply:
What previous code?

Reply:
I mean `int res` is referred from other shell command.

Reply:
`res` is not used within `bt.c` - Might be in the classic shell module :)

Reply:
You are right. It seams `bredr.c` has not be maintained. I will create a PR this issue.